Lib/fontTools/misc/py23.py

```diff
@@ -336,8 +336,19 @@ def round2(number, ndigits=None):
 
 			return float(d)
 
-	# in Python 3, 'round3' is an alias to the built-in 'round'
-	round = round3 = round
+	if sys.version_info[:2] >= (3, 6):
+		# in Python 3.6, 'round3' is an alias to the built-in 'round'
+		round = round3 = round
+	else:
```

Comment:
Looking at https://bugs.python.org/issue27936 it seems the issue only affects 3.5 (right?) so you could do `if sys.version_info[:2] == (3, 5)`